src/newmagic.cpp

```diff
@@ -1739,6 +1739,32 @@ void cast_health_spell(struct char_data *ch, int spell, int sub, int force, char
       } else
         send_to_char(FAILED_CAST, ch);
       break;
+    case SPELL_FLAME_AURA:
+      if (!check_spell_victim(ch, vict, spell, arg))
+        return;
+
+      if (would_become_killer(ch, vict)) {
```

Comment:
I think you can chop out the PK parts, unless this spell has a harmful effect?

Reply:
Wasn't sure if the intention was to make guard mobs aggro at anyone with a flame aura, in which case, casting it on someone else intentionally to aggro mobs could potentially be abused... But I guess someone could also cast Force 1 Invis on another player and accomplish the same potentially abusive tactic?